spec/acceptance/mod_php_spec.rb

```diff
@@ -121,19 +158,15 @@ class {'apache::mod::php':
       it 'succeeds in puppeting php' do
         apply_manifest(pp, catch_failures: true)
       end
-      if (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemmajrelease') == '16.04') ||
-         (fact('operatingsystem') == 'Debian' && fact('operatingsystemmajrelease') == '9')
-        describe file("#{$mod_dir}/php7.0.conf") do
-          it { is_expected.to contain '# somecontent' }
-        end
-      else
-        describe file("#{$mod_dir}/php5.conf") do
-          it { is_expected.to contain '# somecontent' }
-        end
+
+      describe file(php_config_path) do
+        it { is_expected.to contain '# somecontent' }
       end
     end
 
     context 'provide content and template config file' do
+      # does the following even makes sense? Why do we hardcode the php5 template?
```

Comment:
I agree. If you remove `content` the perhaps it can be a test to override the template parameter but currently it doesn't do that. Now it's a duplicate of the `provide custom config file` test

Reply:
Was originally a comment from bastelfreak, but looking it over I can't see one good reason for it to be there.